modules/monitoring-disabling-cross-project-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-cross-project-alerting-rules-for-user-defined-projects_{context}"]
+= Disabling cross-project alerting rules for user-defined projects
+
+Creating cross-project alerting rules for user-defined projects is enabled by default. Cluster administrators can disable the capability in the `cluster-monitoring-config` config map for the following reasons:
+
+* To prevent user-defined monitoring from overloading the cluster monitoring stack. 
+* To prevent buggy alerting rules from being applied to the cluster without having to identify the rule that causes the issue.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa[]
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` config map in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. In the `cluster-monitoring-config` config map, disable the option to create cross-project alerting rules by setting the `rulesWithoutLabelEnforcementAllowed` value under `data/config.yaml/userWorkload` to `false`:
++
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    userWorkload:
+      rulesWithoutLabelEnforcementAllowed: false
+    # ...
+----
+
+. Save the file to apply the changes.
```

Comment:
(nit) there's no file to save but rather a change to be applied to the configmap object (but it's implicit from the previous step). I'd remove this part.

Reply:
I understand what you mean, however, we use this wording in the documentation every time we are working with the config map. It is used throughout our documentation: https://docs.openshift.com/container-platform/4.17/observability/monitoring/configuring-the-monitoring-stack.html even outside of monitoring ([link of another random example](https://docs.openshift.com/container-platform/4.17/nodes/scheduling/descheduler/nodes-descheduler-configuring.html#nodes-descheduler-installing_nodes-descheduler-configuring)). For that reason, I would leave it there for consistency. I hope you don't mind!

Reply:
> I hope you don't mind!

of course not!